lighthouse-cli/test/cli/run-test.js

```diff
@@ -19,40 +19,67 @@ const fastConfig = {
   },
 };
 
+// Map plugin name to fixture since not actually installed in node_modules/.
+jest.mock('lighthouse-plugin-simple', () => {
+  return require('../../../lighthouse-core/test/fixtures/config-plugins/lighthouse-plugin-simple');
+}, {virtual: true});
+
 const getFlags = require('../../cli-flags').getFlags;
 
 describe('CLI run', function() {
-  it('runLighthouse completes a LH round trip', () => {
-    const url = 'chrome://version';
+  describe('LH round trip', () => {
```

Comment:
split this up as it seems like there are a few different things being tested (and `channel` is about to be another check added to this)